### DISCUSSION TITLE
kulala-ui keymaps

### DISCUSSION THREAD

**Discussion by alanfzf** [mistweaverco/kulala.nvim#321] [Q&A] [answered]
<img width="924" alt="Screenshot 2024-11-27 at 12 50 07 PM" src="https://github.com/user-attachments/assets/b6c23885-f2ed-4df5-afec-ec5f973a0fdd">

Hi, i've noticed the kulala-ui buffer has some keymaps pre-established, is there a way to change them or remove them?

**Comment by gorillamoe** [comment, marked as answer]
@YaroSpace created another awesome PR which got merged. It's now possible to disable and/or customize keymaps.
